ui/src/app/icons/edit/component.tsx

```diff
@@ -0,0 +1,83 @@
+import {AfterViewInit, Component, ElementRef, ViewChild} from '@angular/core';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+interface EditComponentState { style: object; }
+
+const style = {
+  height: 14,
```

Comment:
units?

Reply:
I was following the PR advice here: https://github.com/all-of-us/workbench/pull/1644#discussion_r246774697

> FYI: px is assumed as the default, so this can be written as `fontSize: 21`

I'm not opposed to either approach - would prefer to be consistent in whatever we choose.